transcom/mymove
# MB-15751-pricing import tool should allow for domestic linehaul distances of zero

## [Jira ticket](https://dp3.atlassian.net/browse/MB-15751) for this change

## Summary
This PR fixes issues with the import parser erroring out if it encounters a zero value in a rate for domestic linehaul.

Fixes:

-  Updated model validation to require price millicents to be greater or equal to 0
- Modifies test fixture `fixtures/pricing_template_2019-09-19_fake-data.xlsx` to include zero values and the test output to match in `6_2a_domestic_linehaul_prices_golden.csv`


## Verification Steps for Author and Reviewers

1. Run this command in the mymove folder:  `bin/ghc-pricing-parser` 
2. Build the parser using this command: `make bin/ghc-pricing-parser`
3. Run this command `ghc-pricing-parser --filename pkg/parser/pricing/fixtures/pricing_template_2019-09-19_fake-data.xlsx --contract-code=UNIQUECODE --contract-name="Unique Name"`
4. Use a database client to view the re_domestic_linehaul_prices table
5. Execute this query: `select * from re_domestic_linehaul_prices where price_millicents=0;`
6. Verify the price millicents attribute can be equal to zero.

## Screenshot

Database capture of re_domestic_linehaul_table where price_millicents = 0

These entries should match the changes made in the fake data excel file and the golden csv. 

<img width="1626" alt="Screenshot 2023-05-01 at 3 00 05 PM" src="https://user-images.githubusercontent.com/111006369/235512203-f2afcba5-3314-42d3-9547-7715f9d362f1.png">
